### PR TITLE
Update customer-key-set-up.md

### PR DESCRIPTION
**Pull request by brdesai**
Realized that a previous edit that was made to reflect licensing changes never made it to public. Submitting another PR for the same.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/customer-key-set-up.md [hunk 1]
```diff
@@ -26,7 +26,9 @@ You must set up Azure before you can use Customer Key for Office 365. This topic
   
 ## Before you set up Customer Key
 
-Before you get started, ensure that you have the appropriate licensing for your organization. Customer Key in Microsoft 365 is offered in Office 365 E5 or the Advanced Compliance SKU. To understand the concepts and procedures in this topic, review the [Azure Key Vault](https://docs.microsoft.com/azure/key-vault/) documentation. Also, become familiar with the terms used in Azure, for example, [tenant](https://docs.microsoft.com/previous-versions/azure/azure-services/jj573650(v=azure.100)).
+Before you get started, ensure that you have the appropriate licensing for your organization. Starting April 1, 2020, Customer Key in Office 365 is offered in Office 365 E5, M365 E5, M365 E5 Compliance, and M365 E5 Information Protection & Governance SKUs. Office 365 Advanced Compliance SKU is no longer available for procuring new licenses. Customers with existing Office 365 Advanced Compliance licenses will continue to be supported.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
Before you get started, ensure that you have the appropriate licensing for your organization. Starting April 1, 2020, Customer Key in Office 365 is offered in Office 365 E5, Microsoft 365 E5, Microsoft 365 E5 Compliance, and Microsoft 365 E5 Information Protection & Governance SKUs. Office 365 Advanced Compliance SKU is no longer available for procuring new licenses. Customers with existing Office 365 Advanced Compliance licenses will continue to be supported.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by brdesai** [comment]
@KCCross Can you please have this change merged and published at your earliest convenience? The current licensing information for Customer Key is incorrect and hence needs to be corrected asap. Thank you.

**Comment by KCCross** [comment]
Sure, resolve branch conflicts.

**Comment by brdesai** [comment]
@KCCross The conflict has been resolved and changes have been committed.

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Comment by KCCross** [comment]
More recent PR

**Pull request closed without merging**